src/Player/PlayerProxy.cpp

```diff
@@ -70,10 +116,13 @@ void PlayerProxy::play(const string &strUrlTmp) {
             // 播放成功
             *piFailedCnt = 0; // 连续播放失败次数清0
             strongSelf->onPlaySuccess();
+            strongSelf->setTranslationInfo();
+            strongSelf->_on_connect(strongSelf->_transtalion_info);  
```

Comment:
If we do not call `setOnConnect`, `_on_connect` will be empty, how about call `setOnConnect(nullptr)` when creat `PlayerProxy` ?

Reply:
Do you mean that I initiate _on_connect and _on_disconnect in the constructor with nullptr?

Reply:
No, we just need call `setOnConnect(nullptr)` in `PlayerProxy`'s constructor function.